cli/man/TEMPLATE.1.md.example

```diff
@@ -0,0 +1,69 @@
+% COMMAND(1) Cargill, Incorporated | Splinter Commands
+
+<!--
+This file should be named the full command (with dashes between subcommands)
+with an extension of `1.md`. For example `splinter-cert-generate.1.md` is the
+markdown file that will generated into the man page for
+`splinter generate cert`.
+-->
+NAME
+====
+
+**command-name** — about from command help
```

Comment:
I suggest changing "command-name" to "command-subcommand" to clarify the use of dashes.  

Note: Using dashes here is a change from my earlier recommendation. I originally requested spaces between subcommands, but I have changed my mind after researching how similar commands present this line. I'd like to follow the git model, where the command-subcommand string on this line shows what someone would type to view the man page.

Reply:
Also (minor nitpick): I'd suggest writing "About blurb from command help" to reinforce that the blurb should start with a capital letter.

Reply:
updated